src/components/Pricing/PricingBox.tsx

```diff
@@ -8,12 +8,13 @@ const PricingBox = (props: {
   const { price, duration, packageName, subtitle, children } = props;
 
   return (
-    <div className="w-full">
+
+    <div className="w-full ">
       <div
-        className="wow fadeInUp relative z-10 rounded-md bg-white px-8 py-10 shadow-signUp dark:bg-[#1D2144]"
+        className="wow fadeInUp relative z-10 rounded-md bg-white px-6 py-10 shadow-signUp dark:bg-[#1D2144]"
```

Comment:
remove the dark: css element this is causing the bug when the device is in dark mode or else change the overall UI for dark mode